webrtc-stats.html

```diff
@@ -392,6 +395,7 @@ <h3>
 "codec",
 "inbound-rtp",
 "outbound-rtp",
+"svc-layer",
```

Comment:
we should also add an "SVC mode" value to the video RTP sender, I think.

Reply:
Yes that makes sense!